services/wallet/service.go

```diff
@@ -104,7 +104,7 @@ var (
 	errZPInvalidExp       = errors.New("zebpay: linking info validation failed no exp")
 	errZPInvalidAfter     = errors.New("zebpay: linking info validation failed issued at is after now")
 	errZPInvalidBefore    = errors.New("zebpay: linking info validation failed expired is before now")
-	errZPInvalid          = errors.New("zebpay: linking info validation failed, no kyc")
+	errZPInvalidKyc       = errors.New("zebpay: user kyc did not pass")
```

Comment:
I wish GitHub added an actual `+1` emoji instead of using 👍 for it.

So I am sorry @husobee, I let that one slip through the cracks (deliberately), and @clD11 has caught it. 🙂

Reply:
addressed.